### PR TITLE
Ndvi comparison notebook

### PR DESCRIPTION
**Pull request by NeelamRG**
Ndvi comparison notebook

### CODE REVIEW

File: EDS Compatibility Examples/EDS Compatibility Examples.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
 title: EDS Compatibility Examples
-parent: Unknown
+parent: EDS Documentation
+nav_order: 5
 has_children: true
 ---
 
+Here are some examples of using odc stac to work with EDS data
+
+1. How to use odc to create RGB and GeoTiffs from the API stac items
```
Comment (julius-cheung) [inline EDS Compatibility Examples/EDS Compatibility Examples.md]: @NeelamRG - Suggest you use "odc stac" for additional clarity and standardization of our text

File: EDS Compatibility Examples/EDS Compatibility Examples.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+[RGB And GeoTiff](https://github.com/earthdaily/EDA-Documentation/blob/gh-pages/EDS%20Compatibility%20Examples/venus-git-over-LEFR3.ipynb)
+
+2. How to use the odc stac for basic building detection using building footprint 
+[Building detection](https://github.com/earthdaily/EDA-Documentation/blob/gh-pages/EDS%20Compatibility%20Examples/venus-denver-with-building-model.ipynb)
+
+3. How to use odc to query the sentinel-2 and Venus datasets with same criteria and plot the NDVI comparison
```
Comment (julius-cheung) [inline EDS Compatibility Examples/EDS Compatibility Examples.md]: @NeelamRG - Suggest you use "odc stac" for additional clarity and standardization of our text

File: EDS Compatibility Examples/EDS Compatibility Examples.md [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+# Jupyter Notebooks
+Here are some examples of using odc-stac to work with EDS data
+
+### Set Up
+Note: To run these notebooks, you need to set the environment variables as follows
+
+One way is to create a .env file and add the below lines with respective values
+
+EDS_AUTH_URL ="\<Token authorization URL>"\
+EDS_API_URL="https://api.eds.earthdaily.com/archive/v1/stac/v1" \
```
Comment (julius-cheung) [inline EDS Compatibility Examples/EDS Compatibility Examples.md]: @NeelamRG - Please use api.earthdaily.com

File: EDS Compatibility Examples/EDS Compatibility Examples.md [hunk 4]
```diff
@@ -1,7 +1,43 @@
...
+EDS_CLIENT_ID="\<Client_id>"\
+EDS_SECRET="\<Client Secret>"
+
+Another way is to uncomment the line "load_dotenv()" and instead add the below lines
+
+os.environ['EDS_AUTH_URL'] ="\<Token authorization URL>"\
+os.environ['EDS_API_URL']="https://api.eds.earthdaily.com/archive/v1/stac/v1" \
```
Comment (julius-cheung) [inline EDS Compatibility Examples/EDS Compatibility Examples.md]: @NeelamRG  - Please use api.earthdaily.com

File: EDS Compatibility Examples/EDS Compatibility Examples.md (hunk 2 above)
Comment (NeelamRG) [reply EDS Compatibility Examples/EDS Compatibility Examples.md]: Fixed

File: EDS Compatibility Examples/EDS Compatibility Examples.md (hunk 3 above)
Comment (NeelamRG) [reply EDS Compatibility Examples/EDS Compatibility Examples.md]: Fixed

File: EDS Compatibility Examples/EDS Compatibility Examples.md (hunk 4 above)
Comment (NeelamRG) [reply EDS Compatibility Examples/EDS Compatibility Examples.md]: Fixed

**Review by julius-cheung — APPROVED** [review]
LGTM

**Pull request merged**